app/dashboard/utils.py

```diff
@@ -297,6 +297,9 @@ def build_profile_pairs(bounty):
     """
     profile_handles = []
     for fulfillment in bounty.fulfillments.select_related('profile').all().order_by('pk'):
-        if fulfillment.profile and fulfillment.profile.handle and fulfillment.profile.absolute_url:
+        if fulfillment.profile and fulfillment.profile.handle.strip() and fulfillment.profile.absolute_url:
```

Comment:
should make " " falsy

Reply:
Do we need `.strip()` here?

Reply:
the reason i did that is that this comment ( http://bits.owocki.com/0m450K2E0S3t/Screen%20Shot%202018-03-13%20at%206.01.19%20PM.png ) seems to have whitespace in it.. otherwise how did it pass the `fulfillment.profile and fulfillment.profile.handle and fulfillment.profile.absolute_url` check?